wordpress-mobile/WordPress-Android
# Fix #9561: Remove "Discard Local Changes" in the Editor and in PostPreviewActivity

Fixes #9561

I removed the feature from the editor. To clean up string resources I also removed the feature from `PostPreviewAcitivity` (it's not planned in the original ticket, but I think it's not an issue sinc `PostPreviewAcitivity` will be removed soon via #9729 )

To test in the editor:

- Open the editor.
- Do some changes in the post.
- Open the overflow menu, notice there is no "Discard Local Changes" button.

To test in the preview screen:

- On the post list, on a published post with local changes, 
- Tap the "More" button and pick "Preview".
- Make sure a snackbar is displayed and there is only one button: "Publish".

Update release notes:
